Subject: Linux-RT 2.6.24-rc8-rt1
From: http://www.kernel.org/pub/linux/kernel/projects/rt/
Acked-by: Sven-Thorsten Dietrich <sdietrich@suse.de>
From tsutomu.owa@toshiba.co.jp Mon May 14 17:16:37 2007
Date: Mon, 14 May 2007 17:16:37 +0900
From: Tsutomu OWA <tsutomu.owa@toshiba.co.jp>
To: linuxppc-dev@ozlabs.org, linux-kernel@vger.kernel.org
Cc: mingo@elte.hu, tglx@linutronix.de
Subject: Re: [patch 2/5] powerpc 2.6.21-rt1: dummy functions and export _mcount to compile


add dummy functions save_stack_trace(), early_printk() for now and
export _mcount to compile.

Signed-off-by: Tsutomu OWA <tsutomu.owa@toshiba.co.jp>
-- owa

---
 arch/powerpc/kernel/setup_64.c |   18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

Index: linux-2.6.24-rc8-rt1/arch/powerpc/kernel/setup_64.c
===================================================================
--- linux-2.6.24-rc8-rt1.orig/arch/powerpc/kernel/setup_64.c	2008-01-16 22:27:50.000000000 -0500
+++ linux-2.6.24-rc8-rt1/arch/powerpc/kernel/setup_64.c	2008-01-16 22:27:55.000000000 -0500
@@ -607,3 +607,21 @@ struct ppc_pci_io ppc_pci_io;
 EXPORT_SYMBOL(ppc_pci_io);
 #endif /* CONFIG_PPC_INDIRECT_IO */
 
+#ifdef CONFIG_STACKTRACE
+#include <linux/stacktrace.h>
+void notrace save_stack_trace(struct stack_trace *trace)
+{
+}
+#endif /* CONFIG_STACKTRACE */
+
+#ifdef CONFIG_EARLY_PRINTK
+void notrace early_printk(const char *fmt, ...)
+{
+	BUG();
+}
+#endif /* CONFIG_EARLY_PRINTK */
+
+#ifdef CONFIG_MCOUNT
+extern void _mcount(void);
+EXPORT_SYMBOL(_mcount);
+#endif /* CONFIG_MCOUNT */
